Lib/defcon/objects/image.py

```diff
@@ -57,7 +57,7 @@ def __len__(self):
         #     if glyph.image:
         # fileName is required, so if that isn't defined
```

Comment:
the comment still says "fileName is required" which it isn't any more, is it?

Reply:
"fileName is required, so if that isn't defined return 0" sounds about right, i.e. if there's no fileName it's not a valid image

Reply:
ok. previously "fileName" was always there, but None. Anyway, sgtm